### PR TITLE
Gradle updates

### PR DESCRIPTION
**Pull request by CodeCombiner**

### CODE REVIEW

File: android/config.json [hunk 1]
```diff
@@ -2,11 +2,7 @@
...
     ],
-    "libraries": [
-    ],
-    "jars": [
-        "google-basement.jar"
-    ],
     "injectionXML": "manifest.xml",
-	"injectionXSL": "manifest.xsl"
+	"injectionXSL": "manifest.xsl",
+	"injectionTealeafModuleGradleXML": "gradletealeaf.xml"
```
Comment (rampr) [inline android/config.json]: indentation issue

File: android/UtilsPlugin.java [hunk 2]
```diff
@@ -33,380 +38,412 @@
...
-		} catch (PackageManager.NameNotFoundException e) {
-			logger.log("{utils-native} app not available for package");
-		} catch (JSONException e) {
-			logger.log("{utils-native} isAppInstalled json exception" + e);
-		}
-
-		EventQueue.pushEvent(new AppFound(available));
-	}
-}
+    Context _context;
```
Comment (rampr) [inline android/UtilsPlugin.java]: We prefer 2 spaces when using spaces.
Can you please change it to 2 spaces. Thank you

**Comment by CodeCombiner** [comment]
Moved to 2 spaces identation

File: android/config.json [hunk 3]
```diff
@@ -1,12 +1,8 @@
...
-        "UtilsPlugin.java"
-    ],
-    "libraries": [
-    ],
-    "jars": [
-        "google-basement.jar"
-    ],
-    "injectionXML": "manifest.xml",
-	"injectionXSL": "manifest.xsl"
+	"copyFiles": [
```
Comment (rampr) [inline android/config.json]: this file still has tabs. everything else looks good

Comment (CodeCombiner) [reply android/config.json]: Should config files also have 2 spaces?

Comment (rampr) [reply android/config.json]: yeah. But Let's ignore for now in the interest of closing these

**Pull request merged**